.github/workflows/master.yml

```diff
@@ -0,0 +1,63 @@
+# Push to master workflow.
+#
+# Runs when a PR has been merged to the master branch.
+#
+# 1. Generates a release build.
+# 2. If the last commit is a version change, publish.
+
+name: Master
+
+on:
+ push:
+  branches:
+   - master
+
+env:
+  RUST_BACKTRACE: 1
+
+jobs:
+  build:
+    name: Build
```

Comment:
Due to the fact that safe-nd doesn't need any build artifacts because we don't deploy anything, and the publish also doesn't need to actually run a build, technically we don't actually even need a build here, unless we just want to verify that it builds. If you were satisfied that was the case in the PR, we would be all set. However, it doesn't really do much harm to do it either.

Reply:
> unless we just want to verify that it builds.

Yeah, basically I wanted to have a README badge that reports the latest build status. This way people who check out the latest code in master will know what to expect when they try to build it locally.

Although this currently doesn't make much sense unless we merge code that doesn't build. For this to be more useful, we could also make this workflow run on a schedule -- this way we can stay on top of things breaking due to e.g. new rustc version. What do you think?